InscryptionAPI/Triggers/CustomTriggerFinder.cs

```diff
@@ -9,21 +10,153 @@ namespace InscryptionAPI.Triggers;
 /// </summary>
 public static class CustomTriggerFinder
 {
+    public static IEnumerator CustomTriggerSequence(this TriggerReceiver receiver, IEnumerator triggerCoroutine)
```

Comment:
Add some simple documentation to all of the public methods you added to this class that don't have them, or make methods meant to be internal into internal methods - just so that anyone implementing a custom trigger interface knows how to use this class.

Reply:
done